prowler/providers/azure/services/iam/iam_custom_role_permits_administering_resource_locks/iam_custom_role_permits_administering_resource_locks.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "azure",
+  "CheckID": "iam_custom_role_permits_administering_resource_locks",
+  "CheckTitle": "Ensure a Custom Role is Assigned Permissions for Administering Resource Locks",
+  "CheckType": [],
+  "ServiceName": "iam",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "high",
+  "ResourceType": "AzureRole",
+  "Description": "Ensure a Custom Role is Assigned Permissions for Administering Resource Locks",
+  "Risk": "Resouce locks administration is a critical task that should be preformed from a custom role with the appropriate permissions. This ensures that only authorized users can administer resource locks.",
+  "RelatedUrl": "",
```

Comment:
We need to include some resources here to give more information and context to users/customers, maybe some url for the Azure documentation or trendmicro / bridgecrew, probably:
- https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/lock-resources?tabs=json
- https://www.trendmicro.com/cloudoneconformity-staging/knowledge-base/azure/AccessControl/resource-lock-custom-role.html